src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace Microsoft.Extensions.Diagnostics.HealthChecks
 {
-    internal class DefaultHealthCheckService : HealthCheckService
+    internal partial class DefaultHealthCheckService : HealthCheckService
```

Comment:
I should be able to try this out - but are we able to apply a sealed qualifier to a partial type? Trying to determine if this change would affect our ability to do this: https://github.com/dotnet/runtime/issues/49944

Reply:
This works:

```C#
partial class Foo { }

internal sealed partial class Foo { }
```